docs/semgrep-appsec-platform/jira.md

```diff
@@ -23,72 +23,140 @@ Object.entries(frontMatter).filter(
+
+* A short text issue type field
+* A paragraph issue type field
+* A drop-down issue type field with the following options: `Low`, `Medium`, `High`, `Critical`
+* A checkbox issue type field with the following options: `Low`, `Medium`, `High`, `Critical`
+
+If you opt for a drop-down or a checkbox issue type field, verify that:
+
+* There are no misspellings
+* No valid options are missing. If your drop-down or checkbox issue type is missing the `Medium` option, Jira cannot create tickets for medium-severity findings.
+
+## Create tickets
+
+### Code
 
-Once a ticket has been created, a link will be present on the right side of the findings page and along the top of an individual finding's details page.
+To create tickets for one or more Code findings, you can use the triage button:
+
+* On the **Findings** page
+* On an individual finding's **Details** page.
+
+When working on the **Findings** page, you can select and create tickets for individual findings or all findings for a given rule.
+
+:::note
+Creating tickets for many findings at once may take some time. Tickets that take longer than 10 seconds to create are shown in the UI once you refresh the page.
+:::
 
 ![Create Jira ticket - Code](/img/jira-code-findings.png)
-**Figure** Code triage flow
+***Figure.*** Code triage flow
+
+Once a ticket has been created, a link appears on the right side of the **Findings** page and along the top of an individual finding's details page.
 
 ![Jira ticket created - Code](/img/jira-code-ticketed.png)
-**Figure** Code ticket created
+***Figure.*** Code ticket created
 
 ### Supply Chain
 
-:::caution Limitations
-Tickets can only be created for Supply Chain findings with reachable usages. They cannot be created for **Always reachable** or **Unreachable** findings.
+:::info
+Only users who’ve worked with previous versions of the Jira integration can create SCA tickets. Semgrep is working on releasing an update that allows all users to create SCA tickets.
 :::
 
-The Supply Chain triage flow creates tickets for all usages of a given vulnerability. Usages can not be selected individually. 
+The Supply Chain triage flow creates tickets for all usages of a given vulnerability. Usages cannot be selected individually.
+
+To create tickets for one or more Supply Chain findings, you can use the triage button on the:
+
+* **Supply Chain > Vulnerabilities** page
+* Individual finding's **Details** page.
 
 ![Create Jira ticket - Supply Chain](/img/jira-ssc-findings.png)
-**Figure** Supply Chain triage flow
+***Figure*.** Supply Chain triage flow
 
 ![Jira ticket created - Supply Chain](/img/jira-ssc-ticketed.png)
-**Figure** Supply Chain ticket created
+***Figure*.** Supply Chain ticket created
+
+:::caution Limitations
+Tickets can only be created for Supply Chain findings with reachable usages. They cannot be created for **Always reachable** or **Unreachable** findings.
```

Comment:
```suggestion
Tickets can only be created for Supply Chain findings with reachable usages, including always reachable and conditionally reachable. They cannot be created for **Always reachable** or **Unreachable** findings.
```
This is probably more controversial. I can at least verify that Always reachable appeared under the Reachable filter in my SSC UI. I couldn't verify conditionally reachable.

Pick one suggestion and discard the other.